tests/late_joiners.py

```diff
@@ -21,7 +21,8 @@
 
 # 256 is the number of most recent messages that PBFT keeps in memory before needing to replay the ledger
 # the rpc requests that are issued for spinning up the network and checking that all nodes have joined,
-# along with TOTAL_REQUESTS, are enough to exceed this limit
+# along with TOTAL_REQUESTS, are enough to exceed this limit (read requests such as get_commit are processed
+# as write requests in pbft)
 TOTAL_REQUESTS = 60
```

Comment:
I'm still confused by this comment. It mentions the `256` limit, but seems to be describing a variable that's set to `60`. What's the relationship?

Reply:
I'll try to rephrase the comment. The limit is actually wrong, it is `64`. What it means is that traditionally pbft will store up to 64 of each message type (pre-prepare/prepare/commit,) and use those to help other replicas catch up by re transmitting those messages. Once that limit has been exceeded then it will append entries from the ledger, which is what we want to test here.